docs/vetomint.md

```diff
@@ -0,0 +1,199 @@
+# Vetomint
+
+## Summary
+
+Vetomint is a proof-of-stake, byzantine fault tolerant consensus algorithm
+designed to meet the special needs of Simperby. As its name suggests, Vetomint
+is a modification of Tendermint. It is designed to support
+displacing a designated consensus leader for arbitrary reasons via voting.
+Moreover, Vetomint is designed with the self-hosted nature of Simperby in mind,
```

Comment:
I understand what you means, but I think the term `lightweight node operation` seems misleading.
It sounds like we are aiming to optimize CPU resource consumed by node operation.

I changed the sentence to the following. How about this?
> Moreover, Vetomint is designed with the self-hosted nature of Simperby in mind,
thereby runs under long timeout intervals
without performance penalty,
so that validators 
can run their nodes only occasionally.

And I also suggest to replace the expression in the protocol overview document to something else.

Reply:
Good to go.